backend/routes/XrplNFTHelper.js

```diff
@@ -0,0 +1,462 @@
+import xrpl from 'xrpl';
+import axios from 'axios';
+import fs  from 'fs'; //used for storage of token metadata in devmode
+
+import XummSdk from 'xumm-sdk';
+
+
+
+const sdk = new XummSdk.XummSdk('d16951f8-6528-4c3f-88e9-37333e383421','e6d8a068-a55b-4e5a-b540-016847d7e376')
+
+
+
+export class XrplNFTHelper { 
+
+   constructor(details){
+
+
+    
+    this.transactionDetails = details;
+    this.clientDetails = "wss://xrplcluster.com/:51233" //nft-devnet *Change as needed
+   // this.clientDetails = "wss://s.altnet.rippletest.net:51233"
+  }
+
+
+
+  /* Mint Single token
+
+  * @Params (required)
+  -TransactionType: this.transactionDetails.TransactionType,
+  -Account: this.transactionDetails.Account,
+  -URI: this.transactionDetails.URI,
+  -Flags: this.transactionDetails.Flags,
+   -NFTokenTaxon: this.transactionDetails.NFTokenTaxon,
+   
+  @returns array of NFTokenID strings
+  * @returns NFTokenID string
+  */
+  async mintToken(){
+    try {
+     
+      
+    const wallet = xrpl.Wallet.fromSeed(this.transactionDetails.Secret) //secret
+    const client = new xrpl.Client(this.clientDetails) 
+    await client.connect()
+    console.log("Connected to server..minting single token.")
+
+  
+    const transactionData = {
+    TransactionType: this.transactionDetails.TransactionType,
+		Account: this.transactionDetails.Account,
+		URI: this.transactionDetails.URI,
+		Flags: this.transactionDetails.Flags,
+		NFTokenTaxon: this.transactionDetails.NFTokenTaxon,
+    }
+    
+    //submit minting transaction
+    const tx = await client.submitAndWait(transactionData,{wallet})
+
+  const result = await client.request({
+    method: "account_nfts",
+    account: this.transactionDetails.Account
+    })

```

Comment:
This will only give up to 200 responses - if there are more than that issued, you'll have to page through the results or you may not get the latest nft. Also, I'm not sure that there's a guaranteed order for how NFTs are returned (at least I didn't see one listed on the docs page for it here: https://xrpl.org/account_nfts.html#account_nfts)